regparser/layer/formatting.py

```diff
@@ -103,6 +103,7 @@ class Formatting(Layer):
                            + r"(?P<lines>([^\n]*\n)+)"
                            + r"```")
     subscript_re = re.compile(r"([a-zA-Z0-9]+)_\{(\w+)\}")
+    dashes_re = re.compile(r"_{5}$")
```

Comment:
This might be a tad more robust if it were `"_{5,}"` (i.e. five or more). I'm imaging a scenario where the text of the document ends with underscores already (e.g. printer error), leading to multiple sets of five underscores getting matched.